crates/binstalk/src/ops/resolve/resolution.rs

```diff
@@ -139,7 +134,7 @@ impl ResolutionSource {
             .unwrap_or_else(|| Cow::Borrowed(OsStr::new("cargo")));
 
         debug!(
-            "Running `{} install {name} --version {version} --target {target}`",
+            "Running `{} install {name} --version {version}`",
```

Comment:
Wait actually I think there's a subtlety here: if --target is given on the cli we should still pass it to cargo install!

Reply:
Thanks, I forgot this.
Would add it back.

Reply:
I've fixed it now.